src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![doc = include_str!("../README.md")]
+#![cfg_attr(docsrs, feature(doc_auto_cfg))]
```

Comment:
This is a nightly only feature, so it needs to be generally disabled and only enabled when passing the specific flag. Makes sure that this only runs when you're in the context of a docsrs session.